context/src/main/java/com/c8y/ms/templates/context/service/MeasurementService.java

```diff
@@ -0,0 +1,32 @@
+package com.c8y.ms.templates.context.service;
+
+import com.cumulocity.microservice.subscription.service.MicroserviceSubscriptionsService;
+import com.cumulocity.rest.representation.measurement.MeasurementRepresentation;
+import com.cumulocity.sdk.client.measurement.MeasurementApi;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class MeasurementService {
+
+    @Autowired
+    @Qualifier("userMeasurementApi")
+    MeasurementApi userMeasurementApi;
+
+    @Autowired
+    MeasurementApi tenantMeasurementApi;
+
+    @Autowired
```

Comment:
The MicroservceSubscriptionsService can be removed is not used at this service.

Reply:
removed with latest commit